<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttachPanel</class>
 <widget class="QWidget" name="AttachPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attached files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="attach">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Attach</string>
       </property>
       <property name="statusTip">
        <string>Attach</string>
       </property>
       <property name="whatsThis">
        <string>Attach</string>
       </property>
       <property name="text">
        <string>Attach</string>
       </property>
       <property name="icon">
        <iconset resource="Note.qrc">
         <normaloff>:/Note/attach</normaloff>:/Note/attach</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="run">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Run file</string>
       </property>
       <property name="statusTip">
        <string>Run file</string>
       </property>
       <property name="whatsThis">
        <string>Run file</string>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
       <property name="icon">
        <iconset resource="../Manager/Manager.qrc">
         <normaloff>:/Manager/run</normaloff>:/Manager/run</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="copyToClipboard">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Copy to clipboard</string>
       </property>
       <property name="statusTip">
        <string>Copy to clipboard</string>
       </property>
       <property name="whatsThis">
        <string>Copy to clipboard</string>
       </property>
       <property name="text">
        <string>Copy to clipboard</string>
       </property>
       <property name="icon">
        <iconset resource="../Manager/Manager.qrc">
         <normaloff>:/Manager/clipboard</normaloff>:/Manager/clipboard</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="refreshList">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Refresh list</string>
       </property>
       <property name="statusTip">
        <string>Refresh list</string>
       </property>
       <property name="whatsThis">
        <string>Refresh list</string>
       </property>
       <property name="text">
        <string>Refresh list</string>
       </property>
       <property name="icon">
        <iconset resource="Note.qrc">
         <normaloff>:/Note/refresh</normaloff>:/Note/refresh</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="remove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Remove</string>
       </property>
       <property name="statusTip">
        <string>Remove</string>
       </property>
       <property name="whatsThis">
        <string>Remove</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../Manager/Manager.qrc">
         <normaloff>:/fugue-icons/cross</normaloff>:/fugue-icons/cross</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeAll">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Remove all</string>
       </property>
       <property name="statusTip">
        <string>Remove all</string>
       </property>
       <property name="whatsThis">
        <string>Remove all</string>
       </property>
       <property name="text">
        <string>Remove all</string>
       </property>
       <property name="icon">
        <iconset resource="../Manager/Manager.qrc">
         <normaloff>:/Manager/remove_all</normaloff>:/Manager/remove_all</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="listView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listView</tabstop>
  <tabstop>attach</tabstop>
  <tabstop>run</tabstop>
  <tabstop>copyToClipboard</tabstop>
  <tabstop>refreshList</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>removeAll</tabstop>
 </tabstops>
 <resources>
  <include location="../Manager/Manager.qrc"/>
  <include location="Note.qrc"/>
 </resources>
 <connections/>
</ui>
